staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -41,6 +41,18 @@ var (
 		[]string{"verb", "host"},
 	)
 
+	// resolverLatency is a Prometheus Histogram metric type partitioned by
+	// "network", and "address" labels. It is used for the rest client DNS resolver latency metrics.
+	resolverLatency = k8smetrics.NewHistogramVec(
+		&k8smetrics.HistogramOpts{
+			Name:           "rest_client_resolver_duration_seconds",
```

Comment:
A more conventional name for this metric would be `rest_client_dns_resolution_duration_seconds` since we want the thing that we measure to appear in the metric name and in this case we measure dns resolutions. 

This would also make is easier if we want to add error metrics in the future as suggested by @tkashem since we could just name them:
- `rest_client_dns_resolutions_total`
- `rest_client_dns_resolution_errors_total`